pkgs/by-name/dr/dragonflydb/package.nix

```diff
@@ -27,13 +27,13 @@
 
 let
   pname = "dragonflydb";
-  version = "0.1.0";
+  version = "1.29.0";
 
   src = fetchFromGitHub {
     owner = pname;
     repo = "dragonfly";
     rev = "v${version}";
-    hash = "sha256-P6WMW/n+VezWDXGagT4B+ZYyCp8oufDV6MTrpKpLZcs=";
+    hash = "sha256-70e9c82e1db30e8cddde260ccf8c0fb6f41b1ff6e1360b23879f81737948eca7";
```

Comment:
This is not the correct hash. For v1.29.0, it should be `sha256-5147fa9595d03b640403093093aaa690f4d994be`. (But v1.30.0 is out! If you want to use that instead, it would be `sha256-561c0a47338791905f167de6faa12a38bc66a272`.)